imageio/plugins/fits.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, imageio contributors
+# imageio is distributed under the terms of the (new) BSD License.
+
+""" Plugin for reading FITS files.
+"""
+
+from __future__ import absolute_import, print_function, division
+
+from .. import formats
+from ..core import Format
+
+_fits = None  # lazily loaded
+
+
+def load_lib():
+    global _fits
+    from astropy.io import fits as _fits
+    return _fits
+
+
+class FitsFormat(Format):
+
+    """ Flexible Image Transport System (FITS) is an open standard defining a
+    digital file format useful for storage, transmission and processing of
+    scientific and other images. FITS is the most commonly used digital
+    file format in astronomy.
```

Comment:
Should we mention that astropy is needed to use read/write images of this format?